Dockerfile

```diff
@@ -25,24 +25,22 @@ ARG goproxy=https://proxy.golang.org
 ENV GOPROXY=$goproxy
 
 # Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY api/go.mod api/go.mod
-COPY test/go.mod test/go.mod
-COPY go.sum go.sum
-COPY api/go.sum api/go.sum
-COPY test/go.sum test/go.sum
+COPY go.mod go.sum .
+COPY api/go.mod api/go.sum .
```

Comment:
This is now overwriting the root go.mods. `.` means current dir, it works for the root, but for `api` and `test` you need to supply those as targets.

Reply:
Oops. Sorry